chain/chain_config.go

```diff
@@ -104,7 +111,6 @@ func (c *Config) String() string {
 		c.CancunTime,
 		c.PragueTime,
 		engine,
-		c.MordorBlock,
```

Comment:
Must delete the `Mordor: %v}`
https://github.com/0xPolygonHermez/cdk-erigon/pull/85/files#diff-e5f4e248c257170eae82dca72f3cee154dee6f728a0b2804245bb215a5f080f8L86

Reply:
Done!